### PR TITLE
Add user info in Identity page

### PR DESCRIPTION
**Pull request by AlaaElattar**
- #631

### CODE REVIEW

**Comment by AhmedHanafy725** [comment]
@ehab-hassan please have a look at this screen

**Comment by ehab-hassan** [comment]
Group 3 (1)
modifications

If we can use another icon for identity, it would be better. It may be like a guide, Thanks

**Comment by AlaaElattar** [comment]
> Group 3 (1) modifications
> 
> If we can use another icon for identity, it would be better. It may be like a guide, Thanks
- Done

File: app/lib/screens/identity_verification_screen.dart [hunk 1]
```diff
@@ -233,14 +241,69 @@ class _IdentityVerificationScreenState
...
+                                Center(
+                                  child: SizedBox(
+                                    width: MediaQuery.of(context).size.width *
+                                        0.85,
+                                    child: const Divider(
+                                      thickness: 0.5,
+                                      color: Colors.grey,
+                                    ),
+                                  ),
+                                ),
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart]: we can define this divider in a method and reuse it

File: app/lib/screens/identity_verification_screen.dart [hunk 2]
```diff
@@ -514,6 +586,52 @@ class _IdentityVerificationScreenState
     );
   }
 
+  Future copySeedPhrase() async {
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart:570]: This method is copied from the preferences screen, but it's still there. Do you still need it there?

File: app/lib/screens/identity_verification_screen.dart [hunk 3]
```diff
@@ -514,6 +586,52 @@ class _IdentityVerificationScreenState
...
+
+    const seedCopied = SnackBar(
+      content: Text('Seed phrase copied to clipboard'),
+      duration: Duration(seconds: 1),
+    );
+
+    ScaffoldMessenger.of(context).showSnackBar(seedCopied);
+  }
+
+  void _showPhrase() async {
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart:581]: the same

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**